app/src/main/java/com/nexenio/bleindoorpositioningdemo/ui/beaconview/chart/BeaconLineChart.java

```diff
@@ -29,6 +29,10 @@
 
 public class BeaconLineChart extends BeaconChart {
 
+    public static final long MINIMUM_WINDOW_FREQUENCY = TimeUnit.SECONDS.toMillis(10);
```

Comment:
I'd now assume that this is a frequency value in which the window will be update or called. MINIMUM_WINDOW_LENGTH_FREQUENCY might be more comprehensible.